src/BasicLogAnalyser.php

```diff
@@ -290,8 +297,46 @@ private function parseLogEntry($line) {
 			//don't have their own log columns. Extract them from the query string.
 			$result['php_version'] = null;
 			$result['locale'] = null;
+
+
 			if (!empty($result['query_string'])) {
 				parse_str($result['query_string'], $parameters);
+				
+				if (isset($parameters['cms'])) {
+					$parameters['cms'] = str_replace('=', '', $parameters['cms']);
+					
+					
+					if (strpos($parameters['cms'], 'ClassicPress') !== false) {
+						$result['cms'] = 'CP';
+
+						/**
+						 *
+						 * If ClassicPress is detected, will override the cms version with the param "ver" of the site_url
+						 *
+						 */
+						$parts = parse_url($result['site_url']);						
+						if(isset($parts['query'])){
+							$parts = explode('&', $parts['query']);
+
+							$param = array();
+							foreach ($parts as $key => $value) {
+								$compatible = explode('=', $value)[0];
+								$classic_press_version = explode('=', $value)[1];
```

Comment:
Code style: This project uses `$variableName` and not `$variable_name`, so this should be `$classicPressVersion`.